youtube_dl/utils.py

```diff
@@ -4262,16 +4263,38 @@ def try_get(src, getter, expected_type=None):
                 return v
 
 
-def merge_dicts(*dicts):
+def merge_dicts(*dicts, **kwargs):
```

Comment:
Why `kwargs` and not `*dicts, unblank=True, rev=False`? Is it some py2 compat thing I'm missing?

Reply:
Just so:
```
>>> def foo(*args, kw=None):
  File "<input>", line 1
    def foo(*args, kw=None):
                    ^
SyntaxError: invalid syntax
>>> 
```
Weirdly, you can put the keyword args **before** the `*args`, but then you have to call it like that too: otherwise 2.7 thinks that what you wanted to be `args[0]` is `kw` and complains about a duplicate `kw` argument. I don't have a 2.6 to test.